jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codemodernizer/CodeWhispererCodeModernizerSessionTest.kt

```diff
@@ -279,4 +314,35 @@ class CodeWhispererCodeModernizerSessionTest : CodeWhispererCodeModernizerTestBa
         verify(clientAdaptorSpy, times(4)).getCodeModernizationJob(any())
         verify(clientAdaptorSpy, atLeastOnce()).getCodeModernizationPlan(any())
     }
+
+    @Test
```

Comment:
Should we test the contents or bytes written into the file?

Reply:
Bytes wouldn't necessarily matter if we are directly checking the contents of the file to match the text "overwritten"

Reply:
I want to test the checksum of two different files are different. so for the `uploadArtifactToS3(url: String, fileToUpload: File, checksum: String, kmsArn: String)`, even some attacks want to use the presigned s3 upload url to overwrite customers' code, but since the checksum of the attacks' file is different, so the s3 won't allow the upload.